internal/graph/tuplemapper.go

```diff
@@ -89,12 +91,44 @@ func (n NestedTTUMapper) doMap(t *openfgav1.TupleKey) (string, error) {
 	return t.GetUser(), nil
 }
 
+type ObjectIDMapper struct {
```

Comment:
How is this different from NestedTTUMapper?  Would it make sense to just use NestedTTUMapper?  If the name makes it confusing, perhaps rename NestedTTUMapper to be more generic such as tupleKeyMapper?

Reply:
One returns the `User`, the other returns `Object`

Reply:
Ahhh...got it.